src/printDataWithCountryCodes.ts

```diff
@@ -0,0 +1,19 @@
+import { readFileSync } from 'node:fs'
+import { fetchMyIp } from './fetchMyIp.js'
+import { getAllData } from './ipStore.js'
+import { ipToCountryCode } from './ipToCountryCode.js'
+
+type Config = {
+  geolocationApiUri: string
+}
+
+export const printDataWithCountryCodes = async (
+  { geolocationApiUri }: Config,
+) => {
+  const allData = getAllData()
+
+  for (const ip in allData) {
+    const countryCode = await ipToCountryCode(ip, geolocationApiUri)
```

Comment:
wrong arguments order `ipToCountryCode` is expected the URL first and the IP as second param